### PR TITLE
updated instructions

### PR DESCRIPTION
**Pull request by daveKidd**
@BrityHemming   Hey Brit, I think I did it right this time.  Let me know what you think.  Talk to you soon!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
  * 
- * @instructions
- * Implement a higher-order function called `processFirstItem`.
- * It takes two arguments:
- * @param stringList an array of strings.
- * @param callback function that takes a string as its argument.
- * @returns the result of invoking `callback` with the FIRST element in `stringList`.
+ * Use the higher order function processFirstItem below to do the following:
+ *  1. Receive an array of strings called `stringList`
+ *  2. Receive a function that takes a string as its argument called `callback`
```
Comment (BrityHemming) [inline index.js:8]: Receive a callback function that takes a string as it's argument - doesn't necessarily need to be called callback

File: index.js [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+ *  3. Return the result of invoking `callback` with the FIRST element in `stringList`
```
Comment (BrityHemming) [inline index.js:9]: Return the result of invoking the callback function and passing in the FIRST element in the array as the argument

File: index.js [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
  * 
- * Example of usage of this higher-order function:
- * Invoking `processFirstItem` passing `['foo', 'bar']` and `(str) => str + str`,
- * should return 'foofoo'.
+ * For example: Invoking `processFirstItem` and passing `['foo', 'bar']` to stringList
```
Comment (BrityHemming) [inline index.js:11]: I may just remove this part - it kind of makes it more confusing? what do you think?

File: index.js [hunk 4]
```diff
@@ -1,29 +1,27 @@
...
+ * and passing `function(str){return str+str})` to callback should return 'foofoo'.
```
Comment (BrityHemming) [inline index.js:12]: would remove this as well? Or make it clear that it's a different example?

**Review by BrityHemming — CHANGES_REQUESTED** [review]
Hey Dave - a few comments but mostly looks good

File: index.js (hunk 3 above)
Comment (daveKidd) [reply index.js:11]: let me reword it

**Pull request merged**